routes/v1.js

```diff
@@ -149,37 +228,483 @@ api.declare({
   output:         SCHEMA_PREFIX_CONST + 'get-worker-type-response.json#',
   title:          "Get Worker Type",
   description: [
-    // Document what this method does
+    "Retreive a WorkerType definition"
   ].join('\n')
 }, function(req, res) {
   var ctx         = this;
   var workerType  = req.params.workerType;
 
+  if(!req.satisfies({
+    workerType:       workerType
+  })) {
+    return; // by default req.satisfies() sends a response on failure, so we're done
+  }
+
+  var p = ctx.WorkerType.load({workerType: workerType});
+  
+  p = p.then(function(worker) {
+    return res.reply(worker.json());
+  });
+
+  p = p.catch(function(err) {
+    errorHandler(err, res, workerType);
+    return err;
+  });
+
+  return p;
+
 });
 
-// List workerTypes
+
+// Delete workerType
+// TODO: send a pulse message that a worker type was removed
+api.declare({
+  method:         'delete',
+  route:          '/worker-type/:workerType',
+  name:           'removeWorkerType',
+  deferAuth:      true,
+  // TODO: Should we have a special scope for workertype removal?
+  scopes:         ['aws-provisioner:get-worker-type:<workerType>'],
```

Comment:
hmm... that's  good point... I see a potential use-case for some automation process that can updated, but not create/remove worker types.

But we can always add another scope set and maintain backwards compatibility, if we want to, so let's just use:
 * `aws-provisioner:manage-worker-type:<workerType>` (view/modiy)
 * `aws-provisioner:get-worker-type:<workerType>` (view-only)
(I'm also open to better names) 

For the get workertype operation the scopeset would then be:
```js
scopes: [
  ['aws-provisioner:get-worker-type:<workerType>'],
  ['aws-provisioner:manage-worker-type:<workerType>']
]
```

For the create/update/remove it would be:
```js
scopes: [
  ['aws-provisioner:manage-worker-type:<workerType>']
]
```

Note, this is stuff we can change relatively easy... without breaking compatibility.